tests/commands/test_new.py

```diff
@@ -0,0 +1,83 @@
+import subprocess
+import os
+import logging
+from pathlib import Path
+
+logging.basicConfig()
+logger = logging.getLogger('onyo')
+logger.setLevel(logging.INFO)
+
+test_dirs = ['simple',
+             's p a c e s',
+             's p a/c e s',
+             'r/e/c/u/r/s/i/v/e',
+             'relative',
+             'one',
+             'two',
+             'three',
+             'overlap/one',
+             'overlap/two',
+             'overlap/three',
+             'very/very/very/deep'
+             ]
+
+
+def populate_test_repo(path):
```

Comment:
Is there a reason why a new repo is generated for each test?

`rm` and `mv` were destructive, and it made it difficult to know what was where. But `new` is additive.

Creating new repos is time consuming, and if the work isn't needed, we should avoid it, for the sake of keeping test runtime down.